gdnative-derive/src/lib.rs

```diff
@@ -164,6 +197,44 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// - `no_editor`
 ///
 /// Hides the property from the editor. Does not prevent it from being sent over network or saved in storage.
+///
+/// ### `#[methods]`
+/// Adds the necessary information to an `impl` block for 
+///
+/// ### `#[export]`
+/// Registers the attributed function signature to be used by Godot.
+/// A valid function signature must 
+/// - `&self` or `&mut self` as its first argument
+/// - `&<T>` or `TRef<T>` where T refers to the type declared in `#[inherit(T)]` attribute as it's second argument.
```

Comment:
I would use "parameter" instead of "argument", an argument is the value provided at call-site.

Why `&<T>` and not `&T`?
Do backticks not take identifiers verbatim, making it unnecessary to escape?

Typo: "its" instead of "it's" 🙂 

And for the 2nd parameter, maybe mention something like: "This is typically called the _owner_."

Reply:
That's a lot of good catches there! Oops on the &<T> I think I was a little sleepy when I was writing that. I'll make the corrections.